### ISSUE TITLE
Performance on WMT14 dataset

### ISSUE THREAD

**Issue by cswhjiang** [OpenNMT/OpenNMT-py#528]
It seems there is not performance report on the commonly-used dataset. I tested on the WMT14 dataset.

Settings:
For the English to German task, the default arguments are used. The dataset is preprocessed with ``normalize-punctuation.perl`` and ``tokenizer.perl`` and the vocabulary size is 50K. 

Training command:
````
python3.6 -u train.py -data data/wmt14/dataset -save_model data/wmt14/demo-model
````

Training log (last few lines):
```
Epoch 13, 64500/64667; acc:  57.58; ppl:   7.99; 5375 src tok/s; 5294 tgt tok/s;  18163 s elapsed
Epoch 13, 64550/64667; acc:  57.87; ppl:   7.89; 5314 src tok/s; 5280 tgt tok/s;  18177 s elapsed
Epoch 13, 64600/64667; acc:  57.28; ppl:   8.11; 5357 src tok/s; 5276 tgt tok/s;  18191 s elapsed
Epoch 13, 64650/64667; acc:  57.58; ppl:   8.07; 5026 src tok/s; 5043 tgt tok/s;  18205 s elapsed
Train perplexity: 8.16465
Train accuracy: 57.3209
Loading valid dataset from data/wmt14/dataset.valid.1.pt, number of examples: 5699
Validation perplexity: 8.64759
Validation accuracy: 56.7115
Decaying learning rate to 0.015625
```

Translate
````
python3.6 translate.py -model data/wmt14/demo-model_acc_56.71_ppl_8.65_e13.pt -src data/wmt14/test.norm-punc.token.en -output data/wmt14/pred.txt -replace_unk -verbose
````

Bleu score on test set:
```
OpenNMT-py-master# perl tools/multi-bleu.perl data/wmt14/test.norm-punc.token.de < data/wmt14/pred.txt

BLEU = 20.94, 53.9/26.9/15.3/9.3 (BP=0.982, ratio=0.982, hyp_len=61940, ref_len=63078)
```

**Comment by LeenaShekhar** [comment]
Were you able to train the model successfully? If yes, could you please share it with me?

**Comment by vince62s** [comment]
@cswhjiang This is not so bad for the default settings.
If you have time, can you do it using either BPE or sentence piece ?

If you want I have the sentencepiece tokenized data here https://s3.amazonaws.com/opennmt-trainingdata/wmt14-ende-sp.tar.gz
along with the model to detokenize.

With 4 layers of 1024 you should get a higher score with RNNs, about 23 Nist or 24 multibleu
(I got 23.2 on the Lua-onmt)

I have also trained the transformer with py-onmt and after 8 epochs, I get a Nist bleu 22.2 (probably 23 or so multibleu)
But I know we are still missing some proper shuffling / bucketing to optimize.

**Comment by cswhjiang** [comment]
@LeenaShekhar  Here is the link: https://www.dropbox.com/s/yjzl7bd4e2bhncp/demo-model_acc_56.71_ppl_8.65_e13.pt.zip?dl=0

**Comment by cswhjiang** [comment]
@vince62s Is the URL correct? I can not download the data. Here is the error message:
````
<Error>
<Code>AccessDenied</Code>
<Message>Access Denied</Message>
<RequestId>47055A694EA71EA1</RequestId>
<HostId>
isEEc2OoNuINoz3gaKINwWaCF4ZvIEd8LCATVh9wQDAwQUlaUXeLsvZSS7oWFdJ72LvLG8O2fJI=
</HostId>
</Error>
````

**Comment by vince62s** [comment]
sorry try again

**Comment by LeenaShekhar** [comment]
I am able to download it now. Thank you @vince62s . 
I trained a basic model on 200k sentences (En-De) for 12 epochs (all default settings except num layers for encoder and decoder set to 1 and using mlp attention), but that does not look very good. Validation set has 10k sentences. Below are the stats:

Train perplexity: 3.29432
Train accuracy: 71.7549
Loading valid dataset from data/demo.valid.1.pt, number of examples: 5718
Validation perplexity: 151.302
Validation accuracy: 36.9686

Link to the data: https://github.com/OpenNMT/IntegrationTesting/tree/master/data
